canonical/ubuntu.com
# Add CRA banners to security and certs pages

_Go live date June 3rd_
## Done

- Add CRA banners to `/security` and `/security/certification` pages

## QA
- Go to demos for `/security`, check that changes are updated as per [copy docs](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit)
- Go to demos for `/security/certification`, check that changes are updated as per [copy docs](https://docs.google.com/document/d/1yEocR1WXQvN_B1L1yBYrG_0D7ikS6QhcOz27sYs-teI/edit)

## Issue / Card

Fixes [WD-11271](https://warthogs.atlassian.net/browse/WD-11271)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-11271]: https://warthogs.atlassian.net/browse/WD-11271?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
